src/dotnet/HQ.Abstractions/Quotes/GetQuotesV1.cs

```diff
@@ -0,0 +1,41 @@
+
+
+using HQ.Abstractions.Common;
+using HQ.Abstractions.Enumerations;
+
+
+namespace HQ.Abstractions.Quotes
+{
+    public class GetQuotesV1
+    {
+        public class Request : PagedRequestV1
+        {
+            public string? Search { get; set; }
+            public Guid? Id { get; set; }
+            public Guid? clientId { get; set; }
+            public SortColumn SortBy { get; set; } = SortColumn.Name;
+            public SortDirection SortDirection { get; set; } = SortDirection.Asc;
+        }
+        public enum SortColumn
+        {
+            ChargeCode,
+            ClientName,
+            Name
+        }
+
+        public class Response : PagedResponseV1<Record>;
+        public class Record
+        {
+            public Guid Id { get; set; }
+            public int QuoteNumber { get; set; }
+            public string? ChargeCode { get; set; }
+            public Guid ClientId { get; set; }
+            public string ClientName { get; set; }
```

Comment:
Because this maps back to a non-nulalble DB join, we should add `=null!` to tell the compiler that the value will be non-null once populated. We should do that for all warnings with the same message.

See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-forgiving